include/SFML/System/Angle.hpp

```diff
@@ -187,6 +187,7 @@ class Angle
 ////////////////////////////////////////////////////////////
 /// \relates Angle
 /// \brief Overload of == operator to compare two angle values
+/// \note Does not automatically wrap the angle value
```

Comment:
Did it do that before?

Reply:
This behavior did not change in this PR.